Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Pale Genes

## Developer changelog:
- Replaced color replace and other operations by 2 genes: `gene_paleness` and `gene_albino`.


# How to test:
Open the Ruler Designer, try to add sick traits like Ill, Lover's Pox, and Albino to characters. Their skin and hair color should be changed.
